frontend/src/shared-components/index.jsx

```diff
@@ -13,3 +13,4 @@ export * from "./fields";
 export * from "./spinner/spinner";
 export * from "./image-tag/image.tag";
 export * from "./error-page-content/error.page.content";
+export * from "./text-area/text.area";

```

Comment:
Stavio sam `Input`, `Select` i `SelectDate` u shared-components/fields/ s obzirom da dele isti stil. 
Vidim da je ovde stil za `TextArea` poseban, ali opet je to "field" komponenta pa kontam da bi bilo bolje da je i ona u tom folderu (ali u svom subfolderu zajedno sa svojim stilom).
Ako se ne slazes, ostavi je ovde, meni ne smeta :)